src/geometry/manifold/times_cont_mdiff_map.lean

```diff
@@ -37,9 +37,9 @@ structure times_cont_mdiff_map :=
 /-- Bundled smooth maps. -/
 @[reducible] def smooth_map := times_cont_mdiff_map I I' M M' ⊤
 
-localized "notation `C^` n `⦃` I `, ` M `; ` I' `, ` M' `⦄` :=
+localized "notation `C^` n `⟮` I `, ` M `; ` I' `, ` M' `⟯` :=
```

Comment:
I found better parentheses (most importantly because unlike `⦃` they seem to have no other uses). They are typed with \[(.

Reply:
The characters `⟮` and `⟯` are used in a global notation declared in [`field_theory.adjoin`](https://github.com/leanprover-community/mathlib/blob/b6ce982/src/field_theory/adjoin.lean#L144). I'm not sure whether this is a problem though.

cc: @pglutz, @fpvandoorn

Reply:
Ops I overlooked that. Yes, I don't think there is too much risk of them being used together and then `C^` should probably help Lean to recognize which is which? I don't really know how notation works underneath